games/r.yaml

```diff
@@ -406,7 +406,7 @@
   lang:
   - Java
   license:
-  - GPL2
+  - GPL3
```

Comment:
please also change the repo to https://sourceforge.net/projects/rescue/

Reply:
Sure